var/spack/repos/builtin/packages/steps/package.py

```diff
@@ -61,7 +61,7 @@ def cmake_args(self):
             "SUPERLU_DIST"
         ]
         for bundle in bundles:
-            args.append("-DUSE_BUNDLE_{0}:BOOL={1}".format(bundle, bundle_enabled))
+            args.append("-DUSE_BUNDLE_{0}:BOOL={1}".format(bundle, benabled))
```

Comment:
```suggestion
        args.extend("-DUSE_BUNDLE_{0}:BOOL={1}".format(bundle, use_bundle)
                    for bundle in bundles)
```
?

Reply:
And delete the previous line. GH doesn't allow to suggest that.

Reply:
done!